sandbox_dns_unix.go

```diff
@@ -204,7 +204,14 @@ func (sb *sandbox) setupDNS() error {
 
 	currRC, err := resolvconf.Get()
 	if err != nil {
-		return err
+		if !os.IsNotExist(err) {
+			return err
+		} else {
```

Comment:
@thaJeztah Sorry I'm not an expert of network. I just test that my PR has a consistent behavior when `/etc/resolv.conf` does not exist or `/etc/resolv.conf` is an empty file in host. I didn't catch what you mean by `a DNS server running on 127.0.0.1`, is this scenario handled specially in codes?

<br>Also @fcrisciani Thanks for code review, CI is :green_heart: now.